Python/quickstarts/monitor-cloud-logging/functions/main.py

```diff
@@ -0,0 +1,84 @@
+# Welcome to Cloud Functions for Firebase for Python!
+# To get started, simply uncomment the below code or create your own.
```

Comment:
Please don't use "simply" (nor for that matter "please" :D )

Reply:
oops, I'll take that whole comment out. it comes from the CLI template: https://github.com/firebase/firebase-tools/blob/e53c50dff3542a8932cbc668ea326defb0118069/templates/init/functions/python/main.py#L2

Reply:
removed